uaclient/util.py

```diff
@@ -477,3 +477,10 @@ def print_package_list(
         )
     )
     print("")
+
+
+def get_user_or_root_log_path() -> str:
+    if we_are_currently_root():
+        return "/var/log/ubuntu-advantage.log"
+    else:
+        return "~/.cache/ubuntu-pro/ubuntu-pro.log"
```

Comment:
I think we can at least define part of this path on the `defaults` module and reuse it here.
This would even be useful for other places where we specify the non-root log path, like the `log` module

Reply:
I think the log module should have a function that returns the non-root log file location - we can probably use that here